File: PizzaShopAdmin/ClientApp/src/dto/Pizza/PriceDto.ts
import { PizzaSize } from './Enums/PizzaSize';
import { DoughThickness } from './Enums/DoughThickness';

export class PriceDto {
    public id: number;
    public size: PizzaSize;
    public doughThickness: DoughThickness;
    public cost: number;
    public weight: number;
    public checked?: boolean;
}

File: PizzaShopAdmin/Dto/Pizza/PriceDto.cs
using PizzaShop.EntityFramework.Entities.Enum;

namespace PizzaShopAdmin.Dto.Pizza
{
    public class PriceDto
    {
        public int Id { get; set; }
        public PizzaSize Size { get; set; }
        public DoughThickness DoughThickness { get; set; }
        public float Cost { get; set; }
        public int Weight { get; set; }
    }
}
